sky/data/storage.py

```diff
@@ -858,6 +861,8 @@ def add_store(self,
           store_type: StoreType; Type of the storage [S3, GCS, AZURE, R2, IBM]
           region: str; Region to place the bucket in. Caller must ensure that
             the region is valid for the chosen store_type.
+          compress_local: boolean; Decides whether we want to compress the
+            filemount before uploaidng to the bucket.
```

Comment:
```suggestion
            filemount before uploading to the bucket.
```
we also only compress on conditions. pls specify in the docstr